backend/src/main/java/org/eclipse/tractusx/semantics/registry/security/CognitoAuthorizationEvaluator.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2024 Draexlmaier Group
+ */
+
+package org.eclipse.tractusx.semantics.registry.security;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.tractusx.semantics.RegistryProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+
+/**
+ * Conito Autneticator We have two client_ids This is necessary because we have
+ * two clients, one for reading data and one for full access
+ * @author peissing
+ */
+public class CognitoAuthorizationEvaluator extends AuthorizationEvaluator
```

Comment:
What exactly do you mean with this comment?

Reply:
Sorry, it got out of order! This is the original comment I am referring to: https://github.com/eclipse-tractusx/sldt-digital-twin-registry/pull/290/files#r1568851962

Reply:
To avoid misunderstandings ;-) There are no new roles for DTR. I'm using the already defined ones to control the access rights. Via Cognito they are just sent in a different way within the JWT token.

Reply:
I agree, your code does not have new roles introduced, but since the PR was created, the registry had a few new roles, therefore after the update, your code would be missing some methods which are there in the DTR code on the `main` branch today. Please see the full list of roles here and notice the last 3 roles which were created since your PR was issued: https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/docs/README.md#authentication--authorization